mongodbatlas/resource_mongodbatlas_cluster_outage_simulation.go

```diff
@@ -0,0 +1,193 @@
+package mongodbatlas
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/mwielbut/pointy"
+	matlas "go.mongodb.org/atlas/mongodbatlas"
+)
+
+const (
+	errorClusterOutageSimulationCreate  = "error starting MongoDB Atlas Cluster Outage Simulation for Project (%s), Cluster (%s): %s"
+	errorClusterOutageSimulationRead    = "error getting MongoDB Atlas Cluster Outage Simulation for Project (%s), Cluster (%s): %s"
+	errorClusterOutageSimulationDelete  = "error ending MongoDB Atlas Cluster Outage Simulation for Project (%s), Cluster (%s): %s"
+	errorClusterOutageSimulationSetting = "error setting `%s` for MongoDB Atlas Cluster Outage Simulation: %s"
+)
+
+func resourceMongoDBAtlasClusterOutageSimulation() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceMongoDBClusterOutageSimulationCreate,
+		ReadContext:   resourceMongoDBAClusterOutageSimulationRead,
+		UpdateContext: resourceMongoDBClusterOutageSimulationUpdate,
+		DeleteContext: resourceMongoDBAtlasClusterOutageSimulationDelete,
+		Schema: map[string]*schema.Schema{
```

Comment:
the `schema.ResourceImporter` is missing

Reply:
As discussed offline, as per [scope](https://docs.google.com/document/d/1BNkLQBESpEFBXnRtI8YGWHNbbKi_bxwTDDnvvVfHFl8/edit?usp=sharing) import and update functionality are non-goals